packages/client/src/generation/generateClient.ts

```diff
@@ -123,17 +123,8 @@ export async function buildClient({
 
   // we only generate the edge client if `--data-proxy` is passed
   if (dataProxy === true) {
-    fileMap[path.join('edge', 'index.js')] = await JS(edgeTsClient, true)
-    fileMap[path.join('edge', 'package.json')] = JSON.stringify(
```

Comment:
Not needed anymore, instead of having a full folder, we just have one file.

Reply:
(Isomorphism is not needed because bundling for the edge means that we always bundle for the browser)

Reply:
Wait, so that means in isomporphic setups we will be bundling the whole client for the browser, right? In previous setup it was only enums, version info and a bunch of stubs, so significantly smaller amount of code. Is that what we want?

Reply:
If you deploy on Cloudflare Workers, it is by default bundled via the `browser` field. The same applies to Vercel Edge Functions, they always chose `browser` field. That means we cannot have such isomorphism, but just a `browser` build, hence the single file. So yes, if you bundle `@prisma/client/edge` into your frontend, you get the non-needed runtime with it, unfortunately. The best way to tackle this would be to have esm modules so that tree-shaking can take place (but that's a bigger project).